packages/angular_devkit/build_angular/src/tools/sass/lexer.ts

```diff
@@ -0,0 +1,267 @@
+/**
```

Comment:
I'm mostly ignoring the lexer since it doesn't look like it changed much. If there's anything in particular I should be looking at, let me know. I recommend moving this refactor into a different commit so the changes to the lexer which support this Sass change show up in the diff more clearly.